stat/distuv/triangle.go

```diff
@@ -79,6 +79,7 @@ func (t Triangle) Median() float64 {
 	if t.c >= (t.a+t.b)/2 {
 		return t.a + math.Sqrt((t.b-t.a)*(t.c-t.a)/2)
 	}
+
```

Comment:
Please revert this blank line insertion.

In general the convention for returning if-else constructs is `if cond { return res }; return altres`, which in other languages would be written as `if cond { return res } else { return altres }` (there are historical reasons for this relating to return value compiler checks that are not relevant in modern Go code, but the convention remains). This means that the alternative branch is part of the stanza, just as is would be if there were an explicit `else`.

Reply:
Done.